### PR TITLE
Game client

### PR DESCRIPTION
**Pull request by utrennik**
Capture

*** Game start and process CLIENT***
* should listen to the game start event from the server
* should receive and store the object with game settings and data
* should navigate to and display the game page (page elements are displayed depending on the client user role 
- in the TOP section the QUIT game button for the DEALER or EXIT game for the USER - emitting corresponding events
- TIMER if enabled
- SELECT/ADD/REMOVE Issues only for the dealer, 
- RUN/RESET round button only for the dealer, - users score before round end only for the DEALER,
- CARDS only for dealerAsPlayer client / member)
* on RUN/RESET round should send to server the issue data and start the timer (if en..
* should send to the server the client voting result
* should receive and store the voting results
* on the DEALER timer end / voting end send the event to server
* when all issues resolved - the QUIT game on the DEALER client changes to END GAME
* on END GAME button click sends the Event and opens RE..
working:@utrennik

### CODE REVIEW

**Review by HelenBassa — APPROVED** [review]

File: client/src/redux/reducers/game-reducer.ts [hunk 1]
```diff
@@ -96,29 +63,138 @@ export const gameReducer = (state = initialState, { type, payload }) => {
...
+
+      //   return {
+      //     ...state,
+      //     issues: newIssues,
+      //     currentIssue: defaultActiveIssue,
+      //   };
+      // }
+
+      // if (isCurrentIssueInArrray) {
+      //   const activeIssueInArray = newIssues.find((issue) => issue.id === state.currentIssue.id);
```
Comment (aksiantsiuk) [inline client/src/redux/reducers/game-reducer.ts]: remove it

File: client/src/redux/reducers/client-reducer.ts [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+    role: config.DEALER,
+  },
   isDealerLobby: false,
 };
 
-export const clientReducer = (state = initialState, { type, payload }) => {
+// const initialState = {
+//   clientUser: null,
+//   isDealerLobby: false,
+// };
```
Comment (aksiantsiuk) [inline client/src/redux/reducers/client-reducer.ts]: the same

File: client/src/components/ui/issueCard/issueCard.tsx [hunk 3]
```diff
@@ -30,36 +32,49 @@ export const IssueCard = ({
...
-    <div title={name}>
-      <Card className={cardClass}>
+    <div className={isGame ? 'clickable' : ''} title={name}>
+      <Card className={cardClass} onClick={handleIssueSelect}>
         <CardHeader
           className="issue-card-header"
-          title={truncateString(name, config.ISSUE_TITLE_MAX_SYMBOLS)}
+          title={truncateString(name, maxSymbols)}
           subheader={truncateString(priority)}
           subheaderTypographyProps={{ variant: 'subtitle1' }}
```
Comment (aksiantsiuk) [inline client/src/components/ui/issueCard/issueCard.tsx:70]: `subtitle1` is a bad name

File: client/src/components/pages/lobby-page/game-settings-section/switch-lobby.tsx [hunk 4]
```diff
@@ -51,7 +65,8 @@ export const SwitchLobby = ({ switchSettings, handleSwitch }: ISwitchLobby) => (
         />
       </div>
     </div>
-    <div className="switch-lobby">
+    {/* <div className="switch-lobby">
+    //TODO: Implement add cards score from the file
```
Comment (aksiantsiuk) [inline client/src/components/pages/lobby-page/game-settings-section/switch-lobby.tsx:69]: remove if it dont needed anymore

**Pull request merged**